test/math/FloatVectorListTest.ooc

```diff
@@ -294,6 +294,48 @@ FloatVectorListTest: class extends Fixture {
 			list free()
 			filtered free()
 		})
+		this add("higher Precision", func {
+			list := FloatVectorList new()
+			list add(2.0f)
+			list add(1.0f)
+			list add(6.0f)
+			list add(4.0f)
+			list add(7.0f)
+			numberOfPointsBetween := 1
+			tempLists := list higherPrecision(numberOfPointsBetween)
+
+			expect(tempLists count == (list count + numberOfPointsBetween * (list count - 1)))
```

Comment:
`expect(tempLists count, is equal to(list count + numberOfPointsBetween * (list count - 1)))`